crates/lightbeam/Cargo.toml

```diff
@@ -11,19 +11,24 @@ keywords = ["webassembly", "wasm", "compile", "compiler", "jit"]
 edition = "2018"
 
 [dependencies]
-smallvec = "1.0.0"
+auto_enums = "0.7"
+capstone = "0.6.0"
+chain-one = "0.1"
```

Comment:
Would it be possible to split out the changes that introduce the dependency on `chain_one` into a separate PR? Changes introducing new external dependencies are something we'd like to consider carefully, and it's difficult to do that when it's bundled together with the rest of the changes here.